Tasks/InstallAppleProvisioningProfileV1/Tests/L0.ts

```diff
@@ -11,7 +11,7 @@ describe('InstallAppleProvisioningProfile Suite', function () {
     after(() => {
     });
 
-    it('Defaults: install from SecureFile', (done: MochaDone) => {
+    it('Defaults: install from SecureFile', (done: any) => {
```

Comment:
These can be changed to `Done` imported from `mocha`

Reply:
@alex-peck Thank you! I've changed this.